src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogTests.cs

```diff
@@ -359,5 +360,13 @@ public void GetEventLogEntriesTest()
                 Assert.True(Helpers.RetryOnWin7(() => eventLog.Entries.Count) >= 0);
             }
         }
+
+        [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void GetEventLogContainsSecurityLogTest()
+        {
+            EventLog[] eventlogs = EventLog.GetEventLogs();
+            Assert.True(eventlogs.Select(t => t.Log).Contains("Security", StringComparer.CurrentCultureIgnoreCase));
```

Comment:
This works on all OS locales?

If so it should probably be ordinal comparison since you expect this precise string.

Reply:
> This works on all OS locales?

Yes it works on OS locales

Reply:
This comparison must be done as ordinal ignore casing and not culture aware comparison.